tests/simulate/test_lammps.py

```diff
@@ -258,6 +280,18 @@ def test_brownian_dynamics(self):
         else:
             h.run(pot, self.directory)
 
+        # Whole number of steps as float
+        brn.steps = 1.0
+        h = relentless.simulate.LAMMPS(init, brn, executable=self.executable)
```

Comment:
You don't need to make new simulation engines to do a new run, remove these extra instances throughout.